src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      class="search-form"
+      method="get"
+      data-qa="big"
+    >
+      <label class="label">
+        <img
+          class="icon"
+          src="./images/Search.svg"
+          alt="search picture">
+        <input
+        type="text"
+        name="search"
+        data-qa="keypress"
+        class="search-input"
+        placeholder="Try &quot;Los Angeles&quot;">
```

Comment:
```suggestion
        <input
          type="text"
          name="search"
          data-qa="keypress"
          class="search-input"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```

Dont' forget aboud correct indents